commonmark-ext-autolink/pom.xml

```diff
@@ -33,4 +33,13 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
```

Comment:
Is it possible to only put this into the `<build><plugins>` of the parent pom instead of every child pom (I'm not sure if that works or not)?

Reply:
This was the only way I could get it working, been a few months since I put this together, so I can't exactly remember variations I tried.

I'll revisit it and let you know what I find